### DISCUSSION TITLE
Is it possible to turn off client side style tag creation?

### DISCUSSION THREAD

**Discussion by dreadsfun** [emotion-js/emotion#2923] [Q&A]
I'm using the  mantine component library in my project (which uses emotion react) and I'm injecting all the required styles on server side from the emotion cache. Currently when the client side entry loads, emotion creates all the style tags (as it should, normally), but i already have all the styles I need from the server. Is it possible to tell emotion to not create any style tags on the client?

**Comment by Andarist** [comment]
It indicates that your hydration is not set up correctly - hard to tell how you should fix it though if you don't share your code anyhow.

**Reply by dreadsfun** [reply]
sorry, what indicates that? I'm just asking if it is possible for emotion to not do anything at all on the client side
